Plandomizer.py

```diff
@@ -348,7 +348,7 @@ def pattern_matcher(self, pattern):
                             self.major_group.append('Small Key Ring (Thieves Hideout)')
                         else:
                             self.major_group.append('Small Key (Thieves Hideout)')
-                    if self.distribution.settings.shuffle_tcgkeys in ['regional', 'overworld', 'any_dungeon', 'keysanity']:
+                    if self.distribution.settings.shuffle_tcgkeys == 'keysanity':
```

Comment:
Could you help me understand why this only applies to keysanity and not the other potential settings options? Everything else looks good to me including the dungeon info menu (tested it), I'm just confused with this bit in particular.

Reply:
This is just making it consistent with hideout keys. I think it's just a design decision to have it work this way for plando authors. Might also have been an oversight when adding the other key shuffle modes. In either case it's best to keep it consistent here and maybe update all of them (and not just chest game) in a separate PR.

Reply:
Cool sounds good just wanted some clarification 👍🏼